src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -252,9 +258,9 @@ public IDelegateBridge getBattleBridge() {
   }
 
   /**
-   * Add bombardment units to battles.
+   * Add bombardment units to battles. Made public for test purposes only
    */
-  private void addBombardmentSources() {
+  public void addBombardmentSources() {
```

Comment:
It looks like the test that calls this method is in the same package.  Can the accessibility just be changed to package-private instead of public?

Also, @DanVanAtta previously introduced me to the `@VisibleForTesting` annotation.  Annotating this method with `@VisibleForTesting` might be more appropriate than a comment to document your intention to increase the method's visibility purely for testing purposes.